blueprints/automation/Standard Irrigation.yaml

```diff
@@ -28,7 +28,7 @@ description: "Standard irrigation for a Zone"
 
 trigger:
   platform: event
-  event_type: smart_irrigation_start
+  event_type: smart_irrigation_start_all_zones
```

Comment:
no you didn't... this should be `[instance]_start_irrigation_all_zones`. Same in other places.

Reply:
It isn't possible to use  [instance] in a blueprint. The only thing I can do is to create a new input text field with default smart_irrigation so the user can adapt it to his own instance name. I don't know how to get this name in an other way.

Reply:
yes, that is what I was thinking.

Reply:
Is there any way to fix the name for the integration? 
Now I guess it's also not possible to use variables in trigger.

Reply:
Or to fire a fixed event?

Reply:
What is possible:

fire an event like `example_event`

The blueprint, but all automations also look like this:

```
trigger_variables:
  my_event: example_event
trigger:
  platform: event
  event_type: "{{ my_event }}"
  variables:
    name: "{{ trigger.event.data.name }}"

```

Reply:
could be shorter:
```

trigger_variables:
  my_event: example_event
trigger:
  platform: event
  event_type: "{{ my_event }}"

```

Reply:
I think it's always smart_irrigation. Try it: install a new version and give it a non default name. The event should still be the same.

Reply:
Allways smart_irrigation_start_all_zones, not [instance]_start_all_zones?

This would mean change [instance]_start_all_zones to smart_irrigation_start_all_zones everywhere?

Reply:
change it to: `smart_irrigation_start_irrigation_all_zones`. That should be it.